src/components/ProductForList.js

```diff
@@ -53,7 +65,9 @@ export const ProductForList = ({ item, handleDeleteAttempt, token }) => {
         <Link to={`/list/${item.name}/`} state={{ product: item }}>
           <button>details</button>
         </Link>
-        <button onClick={handleDeleteAttempt}>delete</button>
+        <button onClick={handleDeleteAttempt} name={item.name}>
```

Comment:
I think the `Attempt` is extra. It would be fine just as `handleDelete`

Reply:
x2